Meta/run.py

```diff
@@ -367,6 +367,11 @@ def set_up_qemu_binary(config: Configuration):
             # reg.exe not found; errors in reg.exe itself do not throw an error.
             config.qemu_kind = QEMUKind.Other
 
+    if config.qemu_kind == QEMUKind.MacOS:
+        # Assume, for now, that HVF is available.
+        # In set_up_virtualization_support, we'll actually check if it's supported.
+        config.virtualization_support = True
+
```

Comment:
this looks like it should work, but it has the disadvantage of putting logic that sets this flag in multiple places. Well, I say that but there's already logic that sets it to false here for 'windows but out of WSL2". either way, the fix I was thinking of would be in `set_up_virtualization_support`, like so:

```python
def set_up_virtualization_support(config: Configuration):
    provided_virtualization_enable = environ.get("SERENITY_VIRTUALIZATION_SUPPORT")
    # The user config always forces the platform-appropriate virtualizer to be used,
    # even if we couldn't detect it otherwise; this is intended behavior.
    if provided_virtualization_enable is not None:
        config.virtualization_support = provided_virtualization_enable == "1"
    else:
        if (config.kvm_usable and config.architecture == Arch.x86_64 and os.uname().machine == Arch.x86_64.value) or (
            config.qemu_kind == QEMUKind.NativeWindows and config.architecture == Arch.x86_64
        ) or (config.qemu_kind == QEMUKind.MacOS):
            config.virtualization_support = True
 ```
 
 It does seem a bit odd to have the hvf check there after this though, so an even better gold star fix might be to move this:
 
 ```python
     if config.virtualization_support:
        available_accelerators = run(
            [str(config.qemu_binary), "-accel", "help"],
            capture_output=True,
        ).stdout
        # Check if HVF is actually available if we're on MacOS
        if config.qemu_kind == QEMUKind.MacOS and (b"hvf" not in available_accelerators):
            config.virtualization_support = False
 ```
 
 into its own flag `hvf_usable` that gets set similar to `kvm_usable`.